src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,198 @@
+        height: root.height
+        property alias searchText: inputText.input.text
+
+        ColumnLayout {
+            id: contentItemColumn
+            anchors.fill: parent
+            spacing: 0
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 63
+                StatusInput {
+                    id: inputText
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.right: parent.right
+                    anchors.rightMargin: 8
+                    anchors.bottom: parent.bottom
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
+            StatusMenuSeparator { Layout.fillWidth: true }
+            Item {
+                Layout.preferredWidth: parent.width
+                Layout.preferredHeight: 58
+                StatusButton {
+                    id: searchOptionsMenuButton
+                    anchors.left: parent.left
+                    anchors.leftMargin: 16
+                    anchors.verticalCenter: parent.verticalCenter
+                    implicitWidth: (contentItemRowLayout.width + 8)
+                    implicitHeight: 32
+
+                    contentItem: Item {
+                        anchors.fill: parent
+                        RowLayout {
+                            id: contentItemRowLayout
+                            anchors.centerIn: parent
+                            spacing: 2
+                            StatusBaseText {
+                                text: "In: "
+                            }
+                            StatusRoundedImage {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                visible: !!root.selectionBadgeImage
+                                image.source: root.selectionBadgeImage
+                            }
+                            StatusLetterIdenticon {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                letterSize: 11
+                                visible: ((root.selectionBadgeImage === "") && (root.selectionBadgePrimaryText !== "Anywhere"))
+                                color: root.selectionBadgeIdenticonColor
+                                name: root.selectionBadgePrimaryText
+                            }
+                            StatusBaseText {
+                                text: root.selectionBadgePrimaryText
+                            }
+                            StatusIcon {
+                                Layout.alignment: Qt.AlignVCenter
+                                visible: !!root.selectionBadgeSecondaryText
+                                color: Theme.palette.baseColor1
+                                icon: "next"
```

Comment:
The size of this icon is incorrect.
<img width="262" alt="Screenshot 2021-08-03 at 23 15 06" src="https://user-images.githubusercontent.com/445106/128087161-bf77afa7-d854-40c3-afb0-0a7c580b28a7.png">

This was designed:

<img width="290" alt="Screenshot 2021-08-03 at 23 15 26" src="https://user-images.githubusercontent.com/445106/128087212-0cfddb9a-55e5-4ffb-9dd5-d8e083e53e49.png">

Reply:
done. however I can't use 4.5 - 7.5 values from figma because it looks tiny, so I increased to 14.5 - 17.5